.ado/templates/prep-and-pack-nuget.yml

```diff
@@ -20,6 +20,11 @@ steps:
       artifact: ReactWindows
       path: $(System.DefaultWorkingDirectory)/ReactWindows
 
```

Comment:
You could add this (like in the next task) to prevent the waste of time in unsigned builds:

```- ${{ if or(eq(parameters.packMicrosoftReactNative, true), eq(parameters.packMicrosoftReactNativeCxx, true), eq(parameters.packMicrosoftReactNativeManaged, true), eq(parameters.packMicrosoftReactNativeManagedCodeGen, true)) }}:```

Reply:
Actually I think I'll retract this one and I'll just make it part of the agents :)